daemon/dtls.c

```diff
@@ -582,6 +582,9 @@ static int try_connect(struct dtls_connection *d) {
 		case SSL_ERROR_WANT_WRITE:
 			if (d->connected) {
 				ilogs(crypto, LOG_INFO, "DTLS data received after handshake, code: %i", code);
+			} else {
+				ilogs(crypto, LOG_INFO, "DTLS handshake not completed yet, current state: %s", 
```

Comment:
This should be `DEBUG` level I would say. Otherwise it would be too noisy to log every time.